cypress/utils/pages.ts

```diff
@@ -0,0 +1,4 @@
+// NOTE: generates a random string of length 7
+export const genRandomString = (): string => {
+  return Math.random().toString(36).substring(7)
```

Comment:
Nit: What do you feel about including special allowed characters as well ah? 
my thought process here is that we dont really care about the randomness of the string, but we do care if any allowed string is rendered to the user in the notification panel correctly. 

Concretely speaking rather than `genRandomString()`, I am suggesting to use
```
export const generateNotificationString = (): string => {
    const specialCharacters = "!@#$%^&*()"
    const otherLang = "文கி"
  return specialCharacters + otherLang + Math.random().toString(36).substring(100) // longer string to test concatenation 
```

then check if the DOM has rendered a similar string

Reply:
sure - do take note that this function isn't actually used by `editPage.spec` which is where those invariants are asserted (that special characters + other languages are shown properly)